modules/installation/pages/pubcloud-requirements.adoc

```diff
@@ -27,14 +27,14 @@ It is important to perform hostname and IP configuration before you set up {prod
 * {productname} Server and Proxy instances need to run in a network configuration that provides you control over DNS entries, but cannot be accessed from the internet at large.
 * Within this network configuration DNS resolution must be provided: `hostname -f` must return the fully-qualified domain name (FQDN).
 * DNS resolution is also important for connecting clients.
-* DNS is dependent on the cloud framework you choose, refer to the cloud service provider documentation for detailed instructions.
+* DNS is dependent on the cloud framework you choose, refer to the cloud provider documentation for detailed instructions.
 * We recommend that you locate software repositories, the server database, and the proxy squid cache on an external virtual disk.
 This prevents data loss if the instance is unexpectedly terminated.
 This section includes instructions for setting up an external virtual disk.
 
 == Network Requirements
 
-On a public cloud service, you must run {productname} within a restricted network, such as VPC private subnet with an appropriate firewall setting.
+On a public cloud instance, you must run {productname} within a restricted network, such as VPC private subnet with an appropriate firewall setting.
```

Comment:
An instance is just a virtual machine that runs within a public cloud service. Changing `service` to `instance` doesn't make the sentence wrong exactly, but it sounds a little odd to me.

Reply:
updated the wording.